docs/getting-started/intro-to-dockstore-tools-and-workflows.rst

```diff
@@ -1,27 +1,28 @@
 Introduction to Workflows, Tools, and Services
 ==============================================
 
+Tools and workflows hosted on Dockstore are computational steps described in languages that make it easy for their reuse in different compute environments. Generally, descriptor languages request that a user include a containerized compute environment (such as a Docker image) that packages up all of the code and dependencies needed to complete the computational step.
```

Comment:
I'd change 
> computational steps

to
> computational programs

because CWL has its own definition of "steps," and 

> Generally, descriptor languages request that a user include a containerized compute environment

to
> Generally, descriptor languages leverage a containerized compute environment

because I think "request that that a user include" may be interpreted as "request that a user create their own and include" when a user can just use someone else's Docker image.

Reply:
I do like that you adding a little bit of text under the top header to provide some context, but doing that means services should be given context too. Since services work quite differently from tools and workflows, I struggled to come up with a quick way to do this without just repeating the definition that appears below "What is a service?"

I guess you could replace the first sentence of this section with (emphasis only to show changed parts):
> Tools, workflows, **and services** hosted on Dockstore are computational **programs** described in languages that make it easy for their reuse in different compute environments.

But I'm not familiar enough with services to be completely sure how to adjust the last sentence on line 6.